src/OpenTelemetry.ResourceDetectors.AWS/Http/ServerCertificateValidationProvider.cs

```diff
@@ -123,7 +123,7 @@ private bool ValidateCertificate(X509Certificate2? cert, X509Chain? chain, SslPo
                 }
             }
 
-            AWSResourcesEventSource.Log.FailedToValidateCertificate(nameof(ServerCertificateValidationProvider), $"Failed to validate certificate due to {chainErrors}");
+            AWSResourcesEventSource.Log.FailedToValidateCertificate(nameof(ServerCertificateValidationProvider), $"Chain Error(s): {chainErrors}");
```

Comment:
doing string interpolation here is considered a bad practice as it'll allocate the string, even if log is disabled. If this code path is just in the startup, then this is okay.

Reply:
I will just put the chain errors for now as it is similar to the previous log.